src/transformers/integrations/peft.py

```diff
@@ -517,3 +518,49 @@ def _dispatch_accelerate_model(
             offload_dir=offload_folder,
             **dispatch_model_kwargs,
         )
+
+    def delete_adapters(self, adapter_names: Union[List[str], str]):
+        """
+        Delete an adapter's LoRA layers from the underlying model.
```

Comment:
Is it really only LoRA or all types of adapters?

Reply:
Right now, I have experimented with LoRA, but ideally it should work for other adapters too